.circleci/config.yml

```diff
@@ -701,13 +688,13 @@ jobs:
       - checkout-code
       - halt-test-check:
           dir_names: "ui"
+      - gcloud-auth-login-oidc-test
       - run:
           name: Build
           working_directory: ui
           command: |
             set +e
             set -v
-            ../.circleci/activate-service-account.sh
```

Comment:
this was the only usage of `activate-service-account.sh` so it can be removed now